examples/deadline/All-In-AWS-Infrastructure-Basic/ts/package.json

```diff
@@ -18,13 +18,13 @@
     "watch": "tsc -w"
   },
   "devDependencies": {
-    "@types/node": "^14.14.10",
-    "aws-cdk": "1.75.0",
-    "ts-node": "^9.0.0",
-    "typescript": "~4.0.5"
+    "@types/node": "^14.14.14",
+    "aws-cdk": "1.78.0",
+    "ts-node": "^9.1.1",
+    "typescript": "~4.1.3"
```

Comment:
I wonder why aws-cdk is not bumping their typescript version from 3.9.7. But as long as it works for us, we should be fine upgrading it.

Reply:
Their codebase is so large I bet they have a lot more breaking changes to work through than we did.